packages/css/src/config/rules.ts

```diff
@@ -1668,9 +1668,15 @@ const rules = {
         layer: CoreLayer.NativeShorthand,
         colored: true,
         transform(value) {
-            if (!/hidden|dotted|dashed|solid|double|groove|ridge|inset|outset/i.test(value)) {
+            const outlineGlobalValuesRegex =
+                /^(?:inherit|initial|revert|revert-layer|unset)$/i
```

Comment:
These values are applicable for all CSS properties but decided to keep it here while it's used only in this place.